### DISCUSSION TITLE
I have combed through a flow chart of rspack's operation, hoping to help you

### DISCUSSION THREAD

**Discussion by xin2017338** [web-infra-dev/rspack#2675] [Show and tell]
![rspack](https://user-images.githubusercontent.com/28481035/230752055-4da842a7-f612-458b-8208-83ab16deea08.png)

**Comment by faga295** [comment]
Hopes [rspack-guide](https://github.com/web-infra-dev/rspack-dev-guide) can provide a similar graph.

**Comment by hyf0** [comment]
Great chart! Thanks for investigating. We would consider adding a similar graph in dev guide.

**Comment by xin2017338** [comment]
I am constantly updating this graph and documenting my learning

https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&layers=1&nav=1&title=%E6%9C%AA%E5%91%BD%E5%90%8D%E7%BB%98%E5%9B%BE.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1ONrAH--4bQCz95QA0Os1m--xmbpTKmb9%26export%3Ddownload

![image](https://user-images.githubusercontent.com/28481035/233531709-02e50923-c4f7-4f54-928f-452ba7824a45.png)

**Reply by xin2017338** [reply]
update diagram
https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&title=rspack.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1ONrAH--4bQCz95QA0Os1m--xmbpTKmb9%26export%3Ddownload

**Comment by xin2017338** [comment]
![rspack drawio](https://user-images.githubusercontent.com/28481035/233838402-f74b32f3-57bf-4b35-9f9e-e12af95c0653.png)
